stackstorm-tuleap/config.schema.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Please also add an example config file (`tuleap.yaml.example`).

See https://github.com/StackStorm-Exchange/stackstorm-sensu/blob/master/sensu.yaml.example for an example.